frontend/public/lib/parsingOperations.js

```diff
@@ -36,3 +41,31 @@ export function getMessageFromUrl(message) {
   if (typeof message === "object") return message;
   else return decodeURIComponent(message).replaceAll("+", " ");
 }
+
+export function getInfoMetadataByType(type, locale) {
+  if (type === "organizations") return getOrganizationInfoMetadata(locale);
+  if (type === "profiles" || type === "members") return profile_info_metadata(locale);
+  if (type === "projects") return getProjectInfoMetadata();
```

Comment:
Nit: Nice. Why is `profile_info_metadata` snake cased, and the others are camel? We should normalize for consistency IMO.